src/Platform/Microsoft.Testing.Platform/Helpers/ExitCodes.cs

```diff
@@ -15,7 +15,6 @@ internal static class ExitCodes
     public const int TestSessionAborted = 3;
     public const int InvalidPlatformSetup = 4;
     public const int InvalidCommandLine = 5;
-    public const int FeatureNotImplemented = 6;
```

Comment:
It may be worth leaving a comment to not re-use the code 6 as it's already documented in https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-platform-exit-codes.

@Evangelink @MarcoRossignoli Was this intended to be used in some cases?

Reply:
@Evangelink @MarcoRossignoli bump

Reply:
Thanks for the bump! I would keep it commented as suggested by @Youssef1313 as it's already public (not sure if someone relies on it).